tests/integration/helpers.py

```diff
@@ -652,9 +652,15 @@ async def wait_for_mongodb_units_blocked(
 
     This is necessary because the MongoDB app can report a different status than the units.
     """
-    for attempt in Retrying(stop=stop_after_delay(timeout), wait=wait_fixed(1), reraise=True):
-        with attempt:
-            await check_all_units_blocked_with_status(ops_test, db_app_name, status)
+    hook_interval_key = "update-status-hook-interval"
```

Comment:
porting over change from VM

Reply:
When I have the time, I need to send the PR on pytest-operator to have this directly in the plugin code